windows/windows.py

```diff
@@ -0,0 +1,198 @@
+        with open("var.txt", "w") as f:
+            f.write(file_data)
+        os.chdir(path)
+
+    # checking for wget package installed
+    # try:
+    #     import wget
+    # except ImportError:
+    #     print("Wget not installed")
+    #     print("Installing Wget")
+    #     install("wget")
+    # finally:
+    #     os.chdir(var_dir)
+    #     with open("var.txt","r") as f:
+    #         file_data = f.read()
+    #         if(re.search("Wget installed : False", file_data)):
+    #             file_data = re.sub("Wget installed : False", "Wget installed : True", file_data)
+        
+    #     with open("var.txt", "w") as f:
+    #         f.write(file_data)
+    #     os.chdir(path)
+    
+    # checking if Chromedriver is available or not
+    if os.path.exists(f"{path}/chromedriver/chromedriver.exe"):
+        pass
+    else:
+        import requests
+        chrome_driver_url = "https://chromedriver.storage.googleapis.com/107.0.5304.62/chromedriver_win32.zip"
+        print("Downloading Chrome Driver")
+        r = requests.get(chrome_driver_url)
+        with open("chromedriver.zip", 'wb') as fd:
+            for chunk in r.iter_content(chunk_size=128):
+                fd.write(chunk)
+
+        print("Extracting Chrome Driver")
+        with zipfile.ZipFile("chromedriver.zip", 'r') as zip_ref:
+            zip_ref.extractall("chromedriver")
+
+        os.remove("chromedriver.zip")
+
+
+def add_its_profile(ssid, key="iiitbbsr"):
+    '''
+    This function is used for adding the network profile for ITS to windows' network profiles when connecting to ITS for the first time
+    '''
+    print(f"Creating new SSID profile for {ssid}")
+    with open(f"{ssid}_profile.xml", "w") as f1, open("profile_template.txt", "r") as f2:
+        for line in f2:
+            f1.write(line)
+
+    with open(f"{ssid}_profile.xml", "r") as f1:
+        data = f1.read()
+        data = data.replace("{SSID}", ssid)
+        data = data.replace("{password}", key)
+
+    with open(f"{ssid}_profile.xml", "w") as f1:
+        f1.write(data)
+
+    print(f"Adding {ssid} to Windows' network profiles.")
+    subprocess.check_output(["netsh", "wlan", "add", "profile", f"filename={ssid}_profile.xml"], shell=True)
+    print(f"Successfully added {ssid} to Windows' network profiles'")
+    
+    # Deleting the network profile file
+    os.remove(f"{ssid}_profile.xml")
+
+def connect():
+    pass
+
+
+
+def check_available_ssids():
+    # Get a list of available SSIDs
+    available_its = []
+    available_its_ssids = []
+    ssids = subprocess.check_output(["netsh", "wlan", "show", "networks", "mode=bssid"])
+    ssids = ssids.decode("ascii")
+    ssids = ssids.split("\r\n\r\n")
+
+    for _ in ssids:
+        if(_.find("ITS") != -1):
+            available_its.append(_.replace(" ", ""))
+
+    for _ in available_its:
+        res = _.split("\r\n")
+        available_its_ssids.append(res[0][6:])
+
+    return available_its_ssids
+
+def algo_windows():
```

Comment:
please complete the algorithm for windows, there is no call for chrome driver , and try to make the whole process hidden